### ISSUE TITLE
Support special setuptools directives in setup.cfg

### ISSUE THREAD

**Issue by zsol** [python-packaging/dowsing#62]
[Setuptools' doc](https://setuptools.pypa.io/en/latest/userguide/declarative_config.html#specifying-values) specifies a bunch of interesting directives that are [used in the wild](https://github.com/pallets/click/blob/f467dd3aefc3f01563e20e649d0d3c3ba978325e/setup.cfg#L3). They need special interpretation

**Comment by thatch** [comment]
Indeed.  In addition to `attr:` there is also `file:` from https://github.com/python-packaging/dowsing/issues/4

I've been putting this off until the api change necessary to replace the `"??"` string with a proper `Unknown` sentinel, where I would also probably replace the string `"attr: foo"` with `Attr("foo")` that has a `.get()`.
